Common/src/main/java/com/blamejared/crafttweaker/impl/script/scriptrun/runner/FormattingScriptRunner.java

```diff
@@ -0,0 +1,67 @@
+package com.blamejared.crafttweaker.impl.script.scriptrun.runner;
+
+import com.blamejared.crafttweaker.api.CraftTweakerAPI;
+import com.blamejared.crafttweaker.api.zencode.scriptrun.IScriptRunInfo;
+import com.blamejared.crafttweaker.platform.Services;
+import org.openzen.zencode.java.logger.ScriptingEngineLogger;
+import org.openzen.zencode.shared.SourceFile;
+import org.openzen.zenscript.codemodel.HighLevelDefinition;
+import org.openzen.zenscript.codemodel.SemanticModule;
+import org.openzen.zenscript.formatter.FileFormatter;
+import org.openzen.zenscript.formatter.ScriptFormattingSettings;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.List;
+
+final class FormattingScriptRunner extends ScriptRunner {
```

Comment:
So we have either format or execute?
In before we could have both in one run iirc.

Though noone probably used it so I'm fine with the change